testem.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
       mode: 'ci',
       args: [
         // --no-sandbox is needed when running Chrome inside a container
-        process.env.TRAVIS ? '--no-sandbox' : null,
+        process.env.CI ? '--no-sandbox' : null,
```

Comment:
generated by https://github.com/ember-cli/ember-cli-update

note that the updater will try to change many things about this file. however, this is configured to work for testing mapbox-gl in a headless environment. so we should be careful with this.